src/backend/joanie/core/models/courses.py

```diff
@@ -219,6 +221,43 @@ def clean(self):
         return super().clean()
 
 
+class CourseProductRelation(BaseModel):
+    """
+    CourseProductRelation model allows to define position of each product on a course.
+    """
+
+    course = models.ForeignKey(
+        to=Course,
+        verbose_name=_("course"),
+        related_name="product_relations",
+        on_delete=models.RESTRICT,
+    )
+    product = models.ForeignKey(
+        to="Product",
+        verbose_name=_("product"),
+        related_name="course_relations",
+        on_delete=models.CASCADE,
+    )
+    organizations = models.ManyToManyField(
+        to=Organization,
+        related_name="product_relations",
+        verbose_name=_("organizations"),
+    )
+    position = models.PositiveSmallIntegerField(
+        _("position of the product on course"), default=0
+    )
```

Comment:
Don't understand the purpose of this field? In Richie we will not be able to sort products through it.

Reply:
ok, I removed it and we can add it later if it becomes necessary!